pkg/controller/machineset/delete_policy.go

```diff
@@ -31,10 +31,17 @@ const (
 
 type deletePriorityFunc func(machine *v1beta1.Machine) deletePriority
 
+// machineDeleteAnnotationKey annotates machines to be delete among first ones
+var machineDeleteAnnotationKey = "machine.openshift.io/cluster-api-delete-machine"
```

Comment:
We should export this. I would like to reference this symbolically from the CA. But it would be better if it came from pkg/apis as it would be cleaner to not have to reference the controller packages as an API consumer.

Reply:
that belongs to a different PR and a different commit to be cherry-picked from upstream